packages/web/hooks/use-export.ts

```
@@ -37,16 +37,62 @@
+            return `- ${text}\n`
+          case 'code_block':
+            return `\`\`\`\n${text}\n\`\`\`\n\n`
+          case 'hr':
+            return '---\n\n'
+          case 'a':
+            return `[${text}](${node.url})`
+          case 'img':
+            return `![${node.alt || ''}](${node.url})\n\n`
+          default:
+            // Handle text formatting
+            let formattedText = text
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)